### PR TITLE
Publishing the preliminary schedule for CHAOSScon 2023 in Vancouver

### PR DESCRIPTION
**Pull request by germonprez**
Signed-off-by: Matt Germonprez <germonprez@gmail.com>

Publishing the preliminary schedule for CHAOSScon 2023 in Vancouver. I suspect there might be changes but this gets the schedule primarily set.

### CODE REVIEW

File: CHAOSScon/2023NA/schedule.md [hunk 1]
```diff
@@ -2,4 +2,14 @@
 
 ### May 9, 2023 from 1:00 pm - 5:30 pm Pacific Daylight Time 
 
-Schedule is coming soon! 
\ No newline at end of file
+| Time | Topic | Speaker |
+| --- | -- | --- |
+| 1:00pm | Check-in, networking, and coffee/tea |
```
Comment (geekygirldawn) [inline CHAOSScon/2023NA/schedule.md:7]:
```suggestion
| 1:00-1:30 | Check-in, networking, and coffee/tea |

```

Comment (geekygirldawn) [reply CHAOSScon/2023NA/schedule.md:7]: This is a total nit, but I think it would be good to be consistent with how we display the times.

**Review by geekygirldawn — COMMENTED** [review]
I think this looks great! I like that you've added a few minutes of changeover between presentations, which is something we sometimes forget to do :)

**Comment by germonprez** [comment]
Thanks @geekygirldawn!

**Pull request merged**